src/main/java/io/seqera/tower/cli/commands/runs/ViewCmd.java

```diff
@@ -149,6 +161,21 @@ protected Response exec() throws ApiException {
                 utilization.put("cpuEfficiency", progress.getWorkflowProgress().getCpuEfficiency());
             }
 
+            if (labels != null && !labels.getLabels().isEmpty()) {
+                general.put(
+                    "labels",
+                    labels.getLabels().stream()
```

Comment:
This logic to convert a list of labels into a string is duplicated in several places. Maybe is better to move it into a shared method at `FormatHelper` class.